example/build.gradle

```diff
@@ -1,5 +1,24 @@
 description = 'Hibernate Reactive Example'
 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.hibernate:hibernate-gradle-plugin:${hibernateOrmVersion}"
+    }
+}
+
+apply plugin: 'org.hibernate.orm'
+
+hibernate {
+    enhance {
+        enableLazyInitialization= true
+        enableDirtyTracking = true
+        enableAssociationManagement = true
```

Comment:
Do you need all these 3 properties enabled?

Reply:
*That* I am not sure of. I *presume* we don't need `enableDirtyTracking`, but if we're going to enable the bytecode processor we might as well? It help, I presume?

And now that I realize what `enableAssociationManagement` is, I think I *don't* want it.